.ci/conformance/tools/dbscan/patcher.py

```diff
@@ -0,0 +1,30 @@
+import re
+import sys
+
+fil_to_path = sys.argv[1]
+dictionary = [('from sklearn.cluster import DBSCAN', 'from wrappers.wrapper_dbscan import DBSCAN'),
+            ('from sklearn.cluster import dbscan', 'from wrappers.wrapper_dbscan import dbscan'),
+            ('from sklearn.utils._testing import assert_array_equal', 'from sklearn.utils.testing import assert_array_equal')]
+
+def path_file(file_name, dictionary):
+    result_str = ''
+
+    f = open(file_name, "r")
+    lines = f.readlines()
+    f.close()
+
+    for to_path_line, path_line in dictionary:
+        for i in range(len(lines)):
+            if to_path_line in lines[i]:
+                lines[i] = path_line + '\n'
+                break
+
+    f = open(file_name, "w")
+    f.writelines(lines)
+    f.close()
+
+
+if __name__ == "__main__":
+    path_file(fil_to_path, dictionary)
+
```

Comment:
If you would like to keep separate patch files, might be at least move common part to centralized module? 

Or alternatively you can make single patcher script that will accept alg as param.